apps/server/src/modules/common-cartridge/service/common-cartridge.mapper.ts

```diff
@@ -192,6 +194,20 @@ export class CommonCartridgeExportMapper {
 		};
 	}
 
+	public mapFileToResource(
+		fileRecord: FileDto,
+		file: Buffer,
+		element?: FileElementResponseDto
+	): CommonCartridgeResourceProps {
+		return {
+			type: CommonCartridgeResourceType.FILE,
+			identifier: createIdentifier(element?.id),
+			title: element?.content.caption && element.content.caption.trim() ? element.content.caption : fileRecord.name,
```

Comment:
how about using an optional chain expression instead of && ?
```suggestion
			title: element?.content.caption?.trim() ? element.content.caption : fileRecord.name,
```